### DISCUSSION TITLE
Cloudflare

### DISCUSSION THREAD

**Discussion by Tazhys** [Paymenter/Paymenter#394] [Ideas]
The ability to change, add, and remove sites from cloudflare from a module something similar to https://marketplace.whmcs.com/product/2146-cloudflare-whmcs-module

**Comment by 65** [comment]
Full function would also include: 
- Create a zone within an account https://developers.cloudflare.com/api/operations/zones-post 
- List DNS https://developers.cloudflare.com/api/operations/dns-records-for-a-zone-list-dns-records 
- CRUD DNS records https://developers.cloudflare.com/api/operations/dns-records-for-a-zone-create-dns-record 
- Ability to Turn Proxy on and off for a domain record
- A small database of preset values (DNS Templates) that can be easily added or appended to a domain (e.g. Google MX records, server IP's etc)
